signer/aws/aws.go

```diff
@@ -25,51 +27,74 @@ import (
 // OpenSearchService AWS OpenSearchService Name
 const OpenSearchService = "es"
 
-// Signer is a interface that will implement opensearchtransport.Signer
+// OpenSearchServiceServerless AWS OpenSearchService Serverless Name
+const OpenSearchServiceServerless = "aoss"
+
+const emptyBodySHA256 = "2cf24dba5fb0a30e26e83b2ac5b9e29e1b161e5c1fa7425e73043362938b9824"
```

Comment:
We could avoid declaring a constant for this and ensuring the sign function works for a `null` body (and produces this value). Probably shorter, but I don't feel too strongly about it.

Reply:
I did that originally but figured this actually saved us some unnecessary calculation.